### PR TITLE
[SEMI-MODULAR] Snake characters now make sliding noises instead of footsteps

### PR DESCRIPTION
**Pull request by JGlitch**
## About The Pull Request

Title
Snake tails now make sliding noises instead of footsteps

## How Is This Good for The Game

Cause snakes dont have foots to make footstep noises

## Changelog

:cl:
fix: sneks make sliding noises, not footsteps
/:cl:

Credit to NullFag for helping me doing the codeing

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/mob/living/carbon/human/human_update_icons.dm [hunk 1]
```diff
@@ -98,7 +98,12 @@
 				applied_style = STYLE_VOX
 			else if((DIGITIGRADE in dna.species.species_traits) && (wear_suit.mutant_variants & STYLE_DIGITIGRADE))
 				applied_style = STYLE_DIGITIGRADE
-
+		if(applied_style == STYLE_TAUR_SNAKE) //i hate this but i don't know how else to do it
+			RemoveElement(/datum/element/footstep)
+			AddElement(/datum/element/footstep, FOOTSTEP_MOB_SNAKE, 0.6, -6)
+		else
+			RemoveElement(/datum/element/footstep)
+			AddElement(/datum/element/footstep, FOOTSTEP_MOB_HUMAN, 0.6, -6)
```
Comment (ChungusGamer666) [inline modular_skyrat/modules/customization/modules/mob/living/carbon/human/human_update_icons.dm:106]: this is giving me migraines

Comment (JGlitch) [reply modular_skyrat/modules/customization/modules/mob/living/carbon/human/human_update_icons.dm:106]: Shut up chungus, you have the worst smelling nut cheese fungus amongus.

**Comment by GoldenAlpharex** [comment]
This doesn't pass CI, gotta fix that if you want this merged.

**Comment by Gandalf2k15** [comment]
Re-PR if you are interested in this still.

**Pull request closed without merging**